substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1373,9 +1373,30 @@ private void addTargetArguments() {
         return result;
     }
 
+    protected static String createVMInvocationArgumentFile(List<String> arguments) {
+        try {
+            Path argsFile = Files.createTempFile("vminvocation", ".args");
+            String joinedOptions = String.join("\n", arguments);
+            Files.write(argsFile, joinedOptions.getBytes());
+            Runtime.getRuntime().addShutdownHook(new Thread() {
```

Comment:
Use `argsFile.toFile().deleteOnExit()` for more compactness.

Reply:
Fixed